src/PortingAssistant.Client.Analysis/Utils/CodeEntityModelToCodeEntities.cs

```diff
@@ -0,0 +1,308 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Codelyzer.Analysis;
+using Codelyzer.Analysis.Model;
+using NuGet.Versioning;
+using PortingAssistant.Client.Model;
+using TextSpan = PortingAssistant.Client.Model.TextSpan;
+
+namespace PortingAssistant.Client.Analysis.Utils
+{
+    public static class CodeEntityModelToCodeEntities
+    {
+        public static List<SourceFileAnalysisResult> AnalyzeResults(
+            Dictionary<string, List<CodeEntityDetails>> sourceFileToCodeEntities,
+            Dictionary<PackageVersionPair, Task<PackageDetails>> packageResults,
+            Dictionary<string, Task<RecommendationDetails>> recommendationResults,
+            Dictionary<string, List<RecommendedAction>> portingActionResults,
+            string targetFramework = "netcoreapp3.1",
```

Comment:
Is there any case we wont pass the target framework when generating the analyze result?

Reply:
I don't think so, but we have a default value. We need to look at the whole process from start to finish to see if this can be removed.